spec/order_spec.rb

```diff
@@ -0,0 +1,56 @@
+require 'order'
+
+describe Order do
```

Comment:
It would be good to add at least one test verifying that the `complete_order` method calls `SMS#send_sms` - ideally you'd inject this dependency as well, for example:
```ruby
class Order
  def initialize(sms)
     @sms = sms 
  end 

  # ... 
  def send_sms  
    @sms.send_sms(message)
  end
```

rather than...
```ruby
def send_sms  
    text = SMS.new
    message = "Thank you! Your order was placed and will be delivered before #{(Time.now + 3600).strftime("%k:%M")}"
    text.send_sms(message)
  end
```

Then you'd be able to inject a double for this dependency in the test. You could then finally assert that the method `send_sms` was called on the double, after `complete_order` was called, using RSpec's expectations on receiving messages: https://relishapp.com/rspec/rspec-mocks/docs/basics/expecting-messages